doc/userguide/upgrade.rst

```diff
@@ -37,6 +37,12 @@ Major changes
 ~~~~~~~~~~~~~
 - Upgrade of PCRE1 to PCRE2. See :ref:`pcre-update-v1-to-v2` for more details.
 
+Security changes
+~~~~~~~~~~~~~~~~
+- Process creation by Suricata is now disabled by default, this can be configured with `security.limit-noproc` (for LUA scripts for instance)
```

Comment:
the feature is disabled by default, but enabled in the new default config. Upgrade users would not get this unless they update their yaml

Reply:
What do we want ?
I would have started with disabled by default, but I remember you were thinking otherwise, right ?

Reply:
I think the behavior makes sense, but the statement here is a bit incomplete. Upgrading users might expect the setting to be enabled, while they still need to make a config adjustment for it to be enabled.

Reply:
Would it help if we rephrased to something like:
"Process creation by Suricata is now disabled by default. This must be configured with `security.limit-noproc`  "
?

Reply:
New version coming